lib/octobox.rb

```diff
@@ -0,0 +1,31 @@
+require 'octobox/configurator'
+
+module Octobox
+  class << self
+    def config
+      @config ||= Configurator.new
+      if block_given?
+        yield @config
+      end
+      @config
+    end
+
```

Comment:
I actually kind of like the separation from the top level module, was thinking ala [`Rails.configuration`](https://github.com/rails/rails/blob/d25fe31c40928712b5e08fe0afb567c3bc88eddf/railties/lib/rails.rb#L42-L45). I'm not terribly passionate on it though, so if you prefer delegating it, 👌